### DISCUSSION TITLE
求助，旋转表格sidewaystable环境不在目录中显示的问题

### DISCUSSION THREAD

**Discussion by Ryo3024** [tuna/thuthesis#843] [Q&A] [answered]
论文中有个表横着比较好看我就用了
\RequirePackage[figuresright]{rotating}这个宏包
以及正文中的
\begin{sidewaystable}
\begin{tabular}
\end{tabular}
\end{sidewaystable}
这个环境，导致表格不在目录中显示，请问如何解决这个问题？

**Comment by Harry-Chen** [comment, marked as answer]
是否在 `sidewaystable` 中提供了 `\caption` 和 `\label`？
